contracts/ScratchNFT.sol

```diff
@@ -92,8 +92,7 @@ abstract contract ScratchNFT is CommonNFT, PrizeTiers  {
                 _ticketId.toString(),
                 "/",
                 claimDone,
-                "&edition=",
-                editionIdFromTicket.toString()
```

Comment:
this change leads to unused local variable `editionIdFromTicket` and also do we need `editions` mapping in this case?

Some test needs to be adjusted

[FAIL. Reason: Assertion failed.] testTokenURI() (gas: 419156)
Logs:
  Error: a == b not satisfied [string]
        Left: 1/false.json
       Right: 1/false&edition=10

Reply:
yea we dont need edition anymore for purpose of tokenURI